microsoft/testsuites/ltp/ltp.py

```diff
@@ -53,8 +53,8 @@ class Ltp(Tool):
     LTP_TESTS_GIT_TAG = "20190930"
     LTP_GIT_URL = "https://github.com/linux-test-project/ltp.git"
     BUILD_REQUIRED_DISK_SIZE_IN_GB = 2
-    LTP_RESULT_PATH = "/opt/ltp/ltp-results.log"
-    LTP_OUTPUT_PATH = "/opt/ltp/ltp-output.log"
+    LTP_RESULT_PATH = "/home/lisatest/ltp-results.log"
+    LTP_OUTPUT_PATH = "/home/lisatest/ltp-output.log"
```

Comment:
Can you please use this from the username variable? `lisatest` name can change

Reply:
Users should have full permissions for their home folder.

Reply:
Sure, Where is the variable store? Lisa.environment?

Reply:
I think it should be present in `RemoteNode.connection_info`

Reply:
Ok thank you